minecraft_servers/cubic/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+    "server_name": "cubic",
+    "nice_name": "Cubic",
+    "direct_ip": "djhase.de",
+    "supported_languages": [
+      "de", "en"
+    ],
+    "social": {
+      "web": "https://djhase.de",
```

Comment:
Website is not working: Error 405

Reply:
the website is under maintenance

Reply:
Then remove

Reply:
Its fixed

Reply:
Now it's a blank page. 
If you don't have a normal functional and informative website, then remove.